tests/test_plugins/test_mode_solver.py

```diff
@@ -213,6 +213,22 @@ def verify_dtype(ms):
         assert dtype == field.dtype
 
 
+def check_ms_reduction(ms):
+
+    ms_red = ms.reduced_simulation_copy
+    print(ms_red.simulation.symmetry)
```

Comment:
do we need to print here? it's fine but our test outputs are already quite long.